signer/signer.go

```diff
@@ -211,26 +220,24 @@ func (cfg *Configuration) GetKeysAndRand() (priv crypto.PrivateKey, pub crypto.P
 		publicKeyBytes []byte
 		unmarshaledPub crypto.PublicKey
 	)
+	rng = cfg.GetRand()
```

Comment:
I wonder. Since you're touching that code, it is worth renaming `getkeysandrand` to `getkeys` and updating the signers to call `getrand` separately?